WebSecurityScanner/synth.metadata

```diff
@@ -33,5 +25,85 @@
         "generator": "bazel"
       }
     }
+  ],
+  "generatedFiles": [
```

Comment:
This is new! Is this for a specific purpose? To ensure files are removed or added appropriately between diffs, perhaps?

Reply:
This was automatically populated when I ran synthtool, nothing to do with the microgenerator changes.